### ISSUE TITLE
Reusing tableview cell 

### ISSUE THREAD

**Issue by FouadAstitou** [ashfurrow/Collection-View-in-a-Table-View-Cell#33]
Hi, 
When scrolling the collectioview inside the cell and then scrolling the tableview, the cell that gets reused shows the the collection view scrolled to the position that i scrolled to in the first cell.

I was assuming that the storedOffsets was for this unwanted behavior. Unfortunately it is not working. Or is the storedOffsets for another issue?

Please help.

**Comment by ashfurrow** [comment]
Hmm, good question. The reused cell should have its offset set to zero (as long as it doesn't haven't a preciously stored offset, see [this line](https://github.com/ashfurrow/Collection-View-in-a-Table-View-Cell/blob/075b8e40c5c516dda2bb3656abd83bec8efc382b/Table%20View%20in%20a%20Collection%20View/ViewController.swift#L24)). I'm not sure what could be causing the issue you're seeing. I'll have a think.

**Comment by danielchristopher1** [comment]
Having this issue as well.

**Comment by ClintonAkins** [comment]
I had the same issue but fixed it by changing 
    tableViewCell.collectionViewOffset = storedOffsets[indexPath.**row**] ?? 0
to
    tableViewCell.collectionViewOffset = storedOffsets[indexPath.**section**] ?? 0
I hope this helps!

**Comment by ashfurrow** [comment]
Thanks for following up with your solution – it sounds like that would work for a multi-section, single-row table view instead of a single-section, multi-row. Is that the case @ClintonAkins?

**Comment by ClintonAkins** [comment]
You're right. This solution is for a mulit-section, single-row UI, like Netfilx, App Store, etc.

**Comment by ashfurrow** [comment]
Fab, thanks again for the follow-up! going to keep this issue open to help people notice :tada:

**Comment by anishparajuli555** [comment]
One of the solution that worked for me was setting the contentOffset as 
   var collectionViewOffset: CGFloat {
        get {
            return productCollectionView.contentOffset.x
        }
        set {
            productCollectionView.setContentOffset(CGPoint(x: newValue, y: 0 ), animated: false)
        }
    }
